docs/a320-simvars-table.md

```diff
@@ -0,0 +1,284 @@
+| A32NX_OVHD_{name}_PB_IS_ON                             | Bool: True when the push button is ON {name}<br/>APU_START<br/>APU_MASTER_SW<br/>ELEC_COMMERCIAL<br/>PNEU_APU_BLEED<br/>EMER_ELEC_GEN_1_LINE                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
**issue**: Items which have some placeholder (e.g. `{name}`) have that placeholder name and the appropriate values listed in odd ways. For example, in:
> Bool: True when the push button is ON {name}

Having `{name}` at the end doesn't make sense.

Reply:
Thanks for the feedback - was already thinking along the same lines. Tried to do it 1:1 first.
Will invest more time to improve - adding column and looking through the placeholders

For Markdown I use "Markdown Navigator Enhanced" for Clion (payware). Not sure this is a good recommendation as it is not free. Will do some research for a free alternative (for vscode). Where would you put that recommendation - in the PR or on the page itself?

Reply:
You could try `takumii.markdowntable` in vscode, it might be good enough.

Extension recommendations can be added to `.vscode/extensions.json`. That way they'll automatically be recommended when a developer opens the project's folder in vscode.

Reply:
Is it helpful to also separate data type from unit? 
Bool, Integer, Float vs. Number, Percent, Celcius, Hz, etc.?
(most do not specify data type currently but unit - e.g. Volts for A32NX_ELEC_{name}_POTENTIAL - I think that is a float)

Reply:
It's probably best to have one column with the unit, as that is what one uses in code:

```javascript
SimVar.GetSimVarValue("L:A32NX_ELEC_APU_GEN_1_POTENTIAL", "Volts")
```

Reply:
@davidwalschots do you think sorting for column name would be helpful? I would say yes as I think there is no real coordinated sorting otherwise.

Reply:
I'm not sure what sorting means? As far as I know GitHub doesn't support sorting columns.

Reply:
Would be a one time sort with the table editor plugin. Right now ot does not seem to have an order at all.

Reply:
I have ordered the simvars I added such that the ELEC simvars are (mostly) together. We should probably categorise those (and also HYD).

Reply:
Alright - will do and within the groups I will sort alphabetically - the names support sorting and grouping anyway nearly automatically

Reply:
I have updated the table:

- added more categories (might have overdone it - feedback welcome)
- categories should be checked by someone with more A320 system knowledge - would be best to stick to the correct naming and system-groups
- added some missing LVARS and documentation

Also added a list of all LVARS (discovered through SPAD.next) for additional documentation in future PRs. The list might already be somewhat helpful for users looking for a specific LVAR.

I would like to recommend the following:

- quickly get this rolled out so that devs can start using this table instead to avoid that the two pages drifting apart. 
- get additional feedback for the categories - there are several ways to structure this - e.g. by system or by place in cockpit
- I'll be happy to continue to work on this to build an excellent documentation for everybody who wants to use hardware to control the plane
- Maybe the team should create a general naming convention and also do a cleanup round to make naming of existing LVARs more consistent and logical (better now then wait even longer as this will break stuff for users who use the LVARs)

Reply:
The old format should definitely be removed as part of this PR. We won't be maintaining two versions of the same thing.